src/pykiso/cli.py

```diff
@@ -231,6 +265,20 @@ def parse_args(self, ctx, args):
     required=False,
     help="use the specified logger class in pykiso",
 )
+@click.option(
+    "--xray-upload",
+    nargs=3,
+    type=click.Tuple([str, str, str]),
+    required=False,
+    help="Upload the test results to Xray. Requires the username, password and url as arguments.",
+)
+@click.option(
+    "--test-execution-id",
+    nargs=1,
+    type=click.STRING,
+    required=False,
+    help="Test execution ID of Xray where to upload the test results. By default, a new test execution ticket is created except if the ID is given.",
+)
```

Comment:
xray should not become an interface of pykiso. The upload should still be done via the tool.